src/sentry/static/sentry/app/components/broadcastModal.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import IconCloseLg from '../icons/icon-close-lg';
+import ConfigStore from '../stores/configStore.jsx';
+import ApiMixin from '../mixins/apiMixin';
+
+const ReleaseAnnouncement = props => {
+  return (
+    <div>
+      <h3>
+        Releases are better with commits
+        {' '}
+        <span className="badge badge-square badge-new">NEW</span>
+      </h3>
+      <p>
+        <img src="./static/sentry/images/onboarding/release-commits-modal.gif" />
```

Comment:
I'm pretty confident that we need to use a helper to construct this url, but I don't know what it is for JS, because this will be wrong for production.